utility/sort/sort.go

```diff
@@ -0,0 +1,15 @@
+package sort
+
+import (
+	"github.com/gojektech/proctor/proctord/jobs/metadata"
+	"sort"
+)
+
+func Procs(procList []metadata.Metadata) {
+	sort.Slice(procList, func(i, j int) bool {
+		if procList[i].Name < procList[j].Name {
```

Comment:
this if else can be shortened to one line:
```
return procList[i].Name < procList[j].Name 
```

Reply:
Done